proto/validator/accounts/v2/web_api.proto

```diff
@@ -200,6 +206,15 @@ message WalletResponse {
     KeymanagerKind keymanager_kind = 2;
 }
 
+message RecoverWalletRequest {
+    string mnemonic = 1;
+    uint64 num_accounts = 2;
+    string wallet_password = 3;
+    string language = 4;
+    string mnemonic25thWord =5 ;
```

Comment:
```suggestion
    string mnemonic_25th_word =5 ;
```

Reply:
So    Mnemonic25ThWord is the result of mnemonic25th_word   and
Mnemonic_25ThWord is the result of  mnemonic_25th_word     
So after the numbers it upper cases the First letter  and after the underscore it does the same.  If faced by _ and a number it keeps the _.  
I'll maintain   mnemonic25th_word  and skip_mnemonic25th_word  .